### PR TITLE
More info to sizing section

### PR DESCRIPTION
**Pull request by bfallonf**
@jeremyeder @timothysc @ahardin-rh 

Added the table and fixed the math error.

I went to put in some info from the tables in the scaling Ref Arch, but it might be too specific, so I stopped at the one table.

Thoughts?

### CODE REVIEW

File: install_config/install/prerequisites.adoc [hunk 1]
```diff
@@ -119,23 +119,36 @@ this during or after installation.
...
+
+[cols="4,2",options="header"]
+|===
+|Host |Sizing Recommendation
+
+|Maximum nodes per cluster |300
+
+|Maximum pods per nodes |110
+
+|Master host resource usage at max pods |0.25 vCPU and 175 MB
```
Comment (timothysc) [inline install_config/install/prerequisites.adoc]: I would not put this detail there.

Comment (bfallonf) [inline install_config/install/prerequisites.adoc]: @timothysc I can't tell what you mean. You mean both the master and node resource usage? Just the master usage? The entire table?

Comment (timothysc) [inline install_config/install/prerequisites.adoc]: Nix this line...

File: install_config/install/prerequisites.adoc [hunk 2]
```diff
@@ -119,23 +119,36 @@ this during or after installation.
...
+
+|Node host resource usage at max pods |0.5 vCPU and 1.5 GB RAM
```
Comment (timothysc) [inline install_config/install/prerequisites.adoc]: Nix this line too.

**Comment by timothysc** [comment]
lgtm

**Comment by ahardin-rh** [comment]
:+1:

**Comment by bfallonf** [comment]
Great. Merging.

**Comment by bfallonf** [comment]
[rev_history]
|link:../install_config/install/prerequisites.html[Installing -> Prerequisites]
|Corrected information in link:../install_config/install/prerequisites.html#sizing-recommendations[Sizing Recommendations] section.
%

**Pull request merged**